app/controllers/sessions_controller.rb

```diff
@@ -3,8 +3,9 @@ def new; end
 
   def create
     user = User.find_by email: params.dig(:session, :email)&.downcase
-    if user&.authenticate params.dig(:session, :password)
+    if user.try :authenticate, params.dig(:session, :password)
```

Comment:
sao lại thay bằng try

Reply:
@ngocvt-0484 em thấy trong vid bảo dùng try sẽ an toàn hơn mà chị

Reply:
@ngocvt-0484 Trong trường hợp này nó là tương đương, nên em dùng thử một lần cho biết ạ